core/GlobalState.cc

```diff
@@ -2480,10 +2480,44 @@ MethodRef GlobalState::lookupStaticInitForClass(ClassOrModuleRef klass, bool all
     return ref;
 }
 
+namespace {
+
+template <typename GS> ClassOrModuleRef getStaticInitRootSingleton(GS &gs, FileRef file) {
+    auto &db = gs.packageDB();
+    if (!db.enabled()) {
+        return core::Symbols::rootSingleton();
+    }
```

Comment:
```suggestion
```

`getPackageNameForFile` returns a MangledName that does not exist if the packager is not enabled, and similarly `isPackage` returns false, so you could remove this if you want.

Reply:
Ah, nice!